sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -201,6 +201,50 @@ public partial class AutoscaleSettingResource : Azure.ResourceManager.ArmResourc
         public virtual Azure.Response<Azure.ResourceManager.Monitor.AutoscaleSettingResource> Update(Azure.ResourceManager.Monitor.Models.AutoscaleSettingPatch patch, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Monitor.AutoscaleSettingResource>> UpdateAsync(Azure.ResourceManager.Monitor.Models.AutoscaleSettingPatch patch, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public partial class AzureMonitorWorkspaceResource : Azure.ResourceManager.ArmResource
```

Comment:
We should avoid using `Azure` in any models and methods, so please use `rename-mapping` to rename it

Reply:
Renamed this and other AzureMonitorWorkspace(...) models in autorest.md by removing "Azure":

AzureMonitorWorkspaceResource: **MonitorWorkspaceResource**
AzureMonitorWorkspaceResourceCollection: **MonitorWorkspaceResourceCollection**
AzureMonitorWorkspaceResourceData: **MonitorWorkspaceResourceData**
AzureMonitorWorkspaceDefaultIngestionSettings: **MonitorWorkspaceDefaultIngestionSettings**
AzureMonitorWorkspaceMetrics: **MonitorWorkspaceMetrics**
AzureMonitorWorkspaceResourcePatch: **MonitorWorkspaceResourcePatch**